packages/@lwc/ssr-compiler/src/estemplate.ts

```diff
@@ -92,6 +91,9 @@ const getReplacementNode = (
             ? replacementNode.every(validateReplacement)
             : validateReplacement(replacementNode))
     ) {
+        const nodeType = Array.isArray(replacementNode)
+            ? replacementNode.map((n) => n.type).join(',')
```

Comment:
```suggestion
            ? `[${replacementNode.map((n) => n.type)}.join(', ')]`
```
Just to make it nicer.